dataactbroker/fileRoutes.py

```diff
@@ -27,6 +27,32 @@ def add_file_routes(app, create_credentials, is_local, server_path):
     @requires_login
     def submit_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
+
+        sess = GlobalDB.db().session
+
+        start_date = request.json.get('reporting_period_start_date')
```

Comment:
I follow how `@app.route("/v1/delete_submission/", methods=['POST'])` above retrieving parameters. Do you want me change this to use one of the two methods? We might need to change `delete_submission/` too in this case?